physionet-django/physionet/views.py

```diff
@@ -217,8 +223,62 @@ def static_view(request, static_url=None):
 
     return render(request, 'about/static_template.html', params)
 
+
 def tutorial_overview(request):
     """
     Temporary content overview
     """
     return render(request, 'about/tutorial_index.html')
+
+
+@login_required
+def event_home(request):
+    """
+    List of events
+    """
+    user = request.user
+    is_instructor = user.has_perm('user.create_events')
+    events = Event.objects.filter(Q(host=user) | Q(participants__user=user))
+    event_form = AddEventForm(user=user)
+
+    url_prefix = 'http://' + request.get_host()
+
+    if request.method == 'POST':
+        event_form = AddEventForm(user=user, data=request.POST)
```

Comment:
Should this check that the "confirm" value has been posted before adding the user to the event?

Reply:
done!